frontend/src/metabase/lib/pulse.js

```diff
@@ -25,7 +25,7 @@ export function channelIsValid(channel, channelSpec) {
       return false;
   }
   if (channelSpec.recipients) {
-    if (!channel.recipients /* || channel.recipients.length === 0*/) {
```

Comment:
Any idea why this was commented out?

Reply:
I tried poking around in the git blame for that line but there wasn't any obvious "comment this out because it breaks the world" type of commit.